src/sentry/web/frontend/setup_wizard.py

```diff
@@ -108,25 +109,24 @@ def get(self, request: HttpRequest, wizard_hash) -> HttpResponse:
             org_mappings_map[mapping.organization_id] = serialized_mapping
 
         keys_map = defaultdict(list)
-        for key in region_data["keys"]:
-            serialized_key = {
-                "dsn": {"public": key.dsn_public},
-                "isActive": key.is_active,
-            }
-            keys_map[key.project_id].append(serialized_key)
-
         filled_projects = []
-
-        for project in region_data["projects"]:
-            enriched_project = {
-                "slug": project.slug,
-                "id": project.id,
-                "status": STATUS_LABELS.get(project.status, "unknown"),
-            }
-            # The wizard only reads the a few fields so serializing the mapping should work fine
-            enriched_project["organization"] = org_mappings_map[project.organization_id]
-            enriched_project["keys"] = keys_map[project.id]
-            filled_projects.append(enriched_project)
+        for region_name, region_data in region_data_map.items():
+            for key in region_data["keys"]:
+                serialized_key = {
+                    "dsn": {"public": key.dsn_public},
+                    "isActive": key.is_active,
+                }
+                keys_map[key.project_id].append(serialized_key)
```

Comment:
I was looking through this code earlier after seeing the issue. It looks like we can potentially flatten/combine a few of these loops that iterate over `region_data_map`. Is there any reason not to just inline this logic with the original `region_data["keys"]` set logic on line 97?